tools/ratbagd.py

```diff
@@ -251,7 +251,7 @@ class Ratbagd(_RatbagdDBus):
             (GObject.SIGNAL_RUN_FIRST, None, (GObject.TYPE_PYOBJECT,)),
     }
 
-    def __init__(self):
+    def __init__(self, verbose=0):
```

Comment:
This seems to be unused?

Reply:
I think I already answered that to @whot. The verbose is only used in `ratbagc.py`, where it makes sense.

With the new lazy evaluation of the devices in `ratbagc.py`, I should be able to get rid of this requirement and clean this one up from `ratbagd.py`